### ISSUE TITLE
Fieldtype Internal Server Error 

### ISSUE THREAD

**Issue by cole007** [ethercreative/seo#249]
### Description

This is possibly quite a niche issue but have started experiencing issues with the SEO plugin for sites where the siteURL is specified as a Protocol relative url, eg `//mysite.com/` rather than `https://mysite.com/`

When this is the case the SEO fieldtype is throwing an internal server error when adding/editing entries that feature this field.

Some debugging seems to isolate this to line 120 of the fieldtype and specifically the `|replace` filter of the fieldtype template:

    <span id="{{ id }}Slug">{{ url ? url|replace(siteUrl, '') : '...' }}</span>

The specific error being thrown in the log is as follows:

    yii\base\ErrorException: preg_replace(): Unknown modifier 'l' in /XXX/deploy/vendor/craftcms/cms/src/web/twig/Extension.php:490

Basically passing a protocol relative URL into the `|replace` filter seems to throw a wobbly as I guess it is interpreting the slashes at the start and end of the string as regex parameters.

Not sure whether possible/desirable to factor this into the fieldtype template or alternatively pass the siteUrl value via the UrlHelper method into `ether/seo/src/fields/SeoField.php` (in which case the necessary protocols are automatically applied), for example:

```
return $craft->view->renderTemplate(
    'seo/_seo/fieldtype',
        [
	     …   
            'siteUrl' => UrlHelper::siteUrl(),
            …
        ]
);

```

### Additional info

- Craft version: 3.3.0.1 
- SEO version: 3.6.2
- PHP version: 7.1.30
- Database driver & version: MySQL 5.7.26
